lighthouse-core/runner.js

```diff
@@ -137,15 +141,25 @@ class Runner {
         configSettings: settings,
         categories,
         categoryGroups: runOpts.config.groups || undefined,
-        timing: {total: Date.now() - startTime},
+        timing: {entries: artifacts.Timing || [], total: 0},
       };
 
+      // Summarize all the timings and drop onto the LHR
+      log.timeEnd(runnerStatus);
+      lhr.timing.entries.push(...log.takeTimeEntries());
```

Comment:
should do `const timing = {entries: [...artifacts.Timing, ...log.takeTimeEntries()], total: 0};` so the artifact isn't altered